automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeDataprocSpec.scala

```diff
@@ -298,7 +298,7 @@ class RuntimeDataprocSpec
       status <- IO.fromOption(DataprocClusterStatus.withNameInsensitiveOption(cluster.getStatus.getState.name))(
         fail(s"Unknown Dataproc status ${cluster.getStatus.getState.name}")
       )
-      _ <- IO(status shouldBe DataprocClusterStatus.Running)
+      _ <- IO(List(DataprocClusterStatus.Running, DataprocClusterStatus.Stopped).contains(status) shouldBe true)
```

Comment:
previous dataproc cluster is always in `RUNNING` status even when stopped, but this is no longer true now that we're inducing "full" stop of a cluster

Reply:
Can we pass in `expectedStatus` to this method so we can verify the specific status?